smallbuffer.txt

```diff
@@ -0,0 +1,15 @@
+dmd/common/string SmallBuffer
```

Comment:
Why was this file created? It's completely unrelated to the PR

Reply:
Wait, it's still here? This should be deleted 😅

Reply:
#13698